### ISSUE TITLE
error reading response to command SMB2_WRITE

### ISSUE THREAD

**Issue by maxciek89** [Sentaroh/SMBSync2#251]
hi
first of all thank you for such light and intuitive app

i have run into a problem - whenever initiating sync action only one files gets copied and i keep getting the following error for the rest

2022/04/13 14:07:32 zdjecia 
Error
Copied:1 Deleted:0 Ignored:0 
Transport0 error reading response to command=SMB2_WRITE,status=0,flags=0x0000,mid=173,wordCount=0,byteCount=33465

I am trying to copy photos from Android phone to OpenMediaVault5 based NAS running on raspberry 4. The error occurs regardless of number of photos in the folders i have tested (from 2 to +1000).

I haven't encountered similar issues when copying multiple files manually from my phone to NAS using other SMB file managers. 

I am accessing NAS without login details but have full read/write privileges enabled for guests.

**Comment by Trubs333** [comment]
Echo above, thanks for a very useful app. I also have this issue copying to rpi4 omv nas but from an orange pi 5b running android 12. Any clues would be appreciated. 

Thanks

**Comment by shrek-sensei** [comment]
hello, i had a similar issue as above and was able to resolve it. 

setup is rpi4 and omv nas. i have read/write enabled for guests and that has worked for other apps (no creds) but got an error when trying to do the same with smbsync2.

i added one of my user creds to the task and the issue went away.

hope this helps someone

**Comment by Trubs333** [comment]
> hello, i had a similar issue as above and was able to resolve it.
> 
> setup is rpi4 and omv nas. i have read/write enabled for guests and that has worked for other apps (no creds) but got an error when trying to do the same with smbsync2.
> 
> i added one of my user creds to the task and the issue went away.
> 
> hope this helps someone

Yes sir it certainly did. Many thanks for taking the time to share 👍😎
